lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -33,22 +33,22 @@ class Core
 
   # Session command options
   @@sessions_opts = Rex::Parser::Arguments.new(
-    "-c"  => [ true,  "Run a command on the session given with -i, or all"             ],
-    "-C"  => [ true,  "Run a Meterpreter Command on the session given with -i, or all" ],
-    "-h"  => [ false, "Help banner"                                                    ],
-    "-i"  => [ true,  "Interact with the supplied session ID"                          ],
-    "-l"  => [ false, "List all active sessions"                                       ],
-    "-v"  => [ false, "List all active sessions in verbose mode"                       ],
-    "-d" =>  [ false, "List all inactive sessions"                                     ],
-    "-q"  => [ false, "Quiet mode"                                                     ],
-    "-k"  => [ true,  "Terminate sessions by session ID and/or range"                  ],
-    "-K"  => [ false, "Terminate all sessions"                                         ],
-    "-s"  => [ true,  "Run a script or module on the session given with -i, or all"    ],
-    "-u"  => [ true,  "Upgrade a shell to a meterpreter session on many platforms"     ],
-    "-t"  => [ true,  "Set a response timeout (default: 15)"                           ],
-    "-S"  => [ true,  "Row search filter."                                             ],
-    "-x" =>  [ false, "Show extended information in the session table"                 ],
-    "-n" =>  [ true,  "Name or rename a session by ID"                                 ])
+    ["-c", "--command"]              => [ true,  "Run a command on the session given with -i, or all"             ],
```

Comment:
Looks like this breaks tab completion

On master, when tab completing `sessions -` you get the available flags
```
smsf6 payload(windows/meterpreter/reverse_tcp) > sessions -<tab><tab>
sessions -C  sessions -S  sessions -d  sessions -i  sessions -l  sessions -q  sessions -t  sessions -v  
sessions -K  sessions -c  sessions -h  sessions -k  sessions -n  sessions -s  sessions -u  sessions -x 
```

On this branch, when tab completing `sessions -` you get no flags:
```
msf6 payload(python/meterpreter/reverse_tcp) > sessions -
```

Interestingly tab completing `jobs -` works just fine